### PR TITLE
186 foursquare relink gdrive

### PR DESCRIPTION
**Pull request by SteveScott**
#186 please review. .env will need to be changed in production. Changes confirmed in dev database.

### CODE REVIEW

File: recipes/foursquare/datacube.py [hunk 1]
```diff
@@ -11,21 +11,24 @@
...
 drive = GoogleDrive(gauth)
 
+
 # Create Engine
 engine = create_engine(os.environ['RDP_DATA'])
 
 # List all files
 def get_date(title:str) -> str:
-    if '2021' not in title:
-        return '2021' + title.replace('.tar.gz', '').replace('data-cube', '')
```
Comment (SteveScott) [inline recipes/foursquare/datacube.py]: The old dates have been back-filled in the new GDrive directory.

**Comment by SteveScott** [comment]
Just wanted to give this a bump. Pushed changes 10 days ago and waiting for approval.

**Review by mgraber — APPROVED** [review]
Looks good to me! @SPTKL would you review as well?

**Comment by SteveScott** [comment]
I changed the .env file.

**Pull request merged**